lib/src/bot.dart

```diff
@@ -97,7 +97,7 @@ class Bot {
     bot.onCommand('write').listen(_writeToCoop);
     bot.onCommand('ping').listen(_ping);
 
-    var bullyMessageRegexp = RegExp(r'эй\,? хуй', caseSensitive: false);
+    var bullyMessageRegexp = RegExp(r'эй.*хуй', caseSensitive: false);
```

Comment:
Feels like it's too wide. It will trigger if there're words in between the key phrase. Let's make sure that it'll count only commas and spaces, not any characters (.*)